### ISSUE TITLE
Make learning rate adaptive to the model in question

### ISSUE THREAD

**Issue by u7122029** [u7122029/bdnn-features#6]
We saw that the ResNet56 model overfitted the dataset by scoring poorly on the test set after roughly epoch 25 of 1200. It is therefore necessary to lower the learning rate to 1e-6 or 1e-7 so that the model uses the entire 1200 epochs to effectively train itself over the dataset without risking overfitting.
